polyphemus/db/migrations/002_create_cursors.rb

```diff
@@ -0,0 +1,14 @@
+Sequel.migration do 
+  up do
+    create_table(:cursors) do
+      primary_key(:id)
+      String :name, unique: true, null: false
+      column :value, :json, null: false
+      DateTime :updated_at, null: false
```

Comment:
Hmm, might these be less generic than "name", "value" - like "job_name", "completion_status" or some such?

Reply:
For name, the name is unique only to the cursor but not the job.  As written, it is possible have multiple cursors per job.  An example for this is a job that processes multiple buckets.  Each bucket would receive different cursors, but a single job class could process each of those separate cursors.  For instance, if we have a common job that applies to all projects.

For value, yeah it's a little ambiguous what name to give it; in general it's just the json payload associated with the cursor.  The idea is that cursors for different types of jobs (such as magma, redcap, metis, or some other source) may have an arbitrary bag of attributes needed to correctly scan.